packages/react/src/primitives/Field/FieldDescription.tsx

```diff
@@ -8,12 +9,15 @@ export const FieldDescription: React.FC<FieldDescriptionProps> = ({
   descriptiveText,
   labelHidden,
   ...rest
-}) => {
-  return !labelHidden && descriptiveText ? (
```

Comment:
We should probably keep the descriptiveText check, because it doesn't make sense to render this div if there is no text to go inside of it.

Reply:
The a11y errors we have on the homepage have  `descriptiveText` `undefined`  and `labelHidden` `false` 😂

Reply:
We only need the descriptiveText to render if there's actualy a string available. That's why I suggested the change below

Reply:
Good catch! Updated